action.yaml

```diff
@@ -11,6 +11,9 @@ inputs:
   context:
     description: The Docker build context directory.
     default: "."
+  dockerfile:
+    description: Path to the Dockerfile.
```

Comment:
what is this path relative to?  repository root?  it's hard to tell from the build-push-action docs...

Reply:
Great question. I'll update the documentation on this. The path is relative to the working directory except for when it is unspecified then it is relative to the context directory